packages/build/package.json

```diff
@@ -127,8 +127,8 @@
   },
   "devDependencies": {
     "@netlify/nock-udp": "^3.1.2",
-    "@opentelemetry/sdk-trace-base": "^1.18.1",
-    "@opentelemetry/api": "^1.7.0",
+    "@opentelemetry/sdk-trace-base": "~1.22.0",
+    "@opentelemetry/api": "~1.8.0",
```

Comment:
we should lock the otel pacakges to the minor version as `1.7.0` is not compatible with `1.8.0`